src/main/java/com/jcabi/beanstalk/maven/plugin/AbstractBeanstalkMojo.java

```diff
@@ -259,6 +269,80 @@ protected boolean isGreen(final Environment env) {
     }
 
     /**
+     * Validate given text in YAML format.
+     *
+     * @param text YAML text
+     * @return Boolean true if valid else false
+     */
+    protected boolean validYaml(final String text) {
+        boolean valid = true;
+        try {
+            new Yaml().load(text);
+        } catch (final YAMLException exception) {
+            valid = false;
```

Comment:
@tmjee does YAMLException provide extended information about issues in the yaml file?
if so it would better inform the user about details instead just say the file is valid or not.
what do you think?